util/status.go

```diff
@@ -25,20 +29,26 @@ func StatusUnchanged(existing, new []v1.LoadBalancerIngress) bool {
 func SliceToStatus(endpoints []string) []v1.LoadBalancerIngress {
```

Comment:
could we use a different name for this method? every time i read Slice to Status, i can't think of what Slice means